code/web/src/modules/user/MyProducts.js

```diff
@@ -0,0 +1,155 @@
+//Imports
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom'
+import { Helmet } from 'react-helmet'
+
+
+// UI Imports
+import { Grid, GridCell } from '../../ui/grid'
+import { H3, H4 } from '../../ui/typography'
+import Button from '../../ui/button'
+import { grey, grey2, black } from '../../ui/common/colors'
+
+//App Imports
+import SubNav from './SubNav'
+import userRoutes from '../../setup/routes/user'
+import { logout } from './api/actions'
+
+class MyProducts extends PureComponent {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      viewAll: true
+    }
+  }
+
+  toggleKept = () => {
+    // event.preventDefault()
+    this.setState({ viewAll: !this.state.viewAll })
+  }
+
+  render() {
+    const kept = true
+    const kept2 = false
+
+    return (
+      <>
+        {/* SEO */}
+        <Helmet>
+            <title>Products - Admin - Crate</title>
+        </Helmet>
+        {/* Top menu bar */}
+        <SubNav />
+        {/* Page Content */}
+        <div>
+          <H3>Delivery History</H3>
+          {/* Top actions bar */}
+          <Grid alignCenter={true} style={{ padding: '1em' }}>
+            <GridCell style={{ textAlign: 'right' }}>
+              <Button theme="primary" style={{ marginTop: '1em' }} onClick={this.toggleKept} >
+                Toggle Kept Products
+              </Button>
+            </GridCell>
+          </Grid>
+          {/* Product list */}
+          <Grid alignCenter={true} style={{ padding: '1em' }}>
+            <GridCell>
+              <table className="striped">
+                <thead>
+                  <tr>
+                    <th>Image</th>
+                    <th>Name</th>
+                    <th>Description</th>
+                    <th>Delivered On</th>
+                    <th style={{ textAlign: 'center' }} >Kept</th>
+                  </tr>
+                </thead>
+
+                {/*This is where we'll map through all state user products and return the tr:*/}
```

Comment:
Thanks for putting this in as a comment, I should've done this as well as the hardcoding of the example rows. This seems solid to me and I'll add the logic of the toggle to this!

Reply:
Rows are going to be great for testing purposes too!